src/main/java/core/basesyntax/classes/figures/Figure.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.classes.figures;
+

```

Comment:
Avoid starting the class implementation with an empty line. Remove this redundant empty line as per the checklist guidelines.